packages/adapter/src/chains/chainData.ts

```diff
@@ -0,0 +1,27 @@
+import type { XToken, XcmConfig } from "@oak-network/config";
+import { XcmInstructionNetworkType } from "../types";
+
+export class ChainData {
+  key: string | undefined;
+
+  assets: XToken[] = [];
+
+  defaultAsset: XToken | undefined;
+
+  endpoint: string | undefined;
+
+  relayChain: string | undefined;
+
+  network: string | undefined;
+
+  paraId: number | undefined;
+
+  ss58Prefix: number | undefined;
+
+  name: string | undefined;
+
+  xcmInstructionNetworkType: XcmInstructionNetworkType =
+    XcmInstructionNetworkType.Null;
+
+  xcm: XcmConfig | undefined;
+}
```

Comment:
This is already defined in `/packages/config/src/chains/types/Chain.ts`. Please import the type from config package.

Besides, the `network` attribute now lives within `XcmConfig`.

Reply:
Yes, eslint requires that we can't have two classes in the same file, so I moved it out.